### ISSUE TITLE
Latest version (1.3.3) fails to install

### ISSUE THREAD

**Issue by noffle** [substack/wit#5]
It isn't obvious to me why, but 1.3.2 installs fine and 1.3.3 does not, despite c72fc47540f021bf6a04ac6e501679468df87ff4 being the only change.

```
[stephen:/tmp]$ npm install wit@1.3.2
wit@1.3.2 node_modules/wit
├── split@0.2.10
├── text-table@0.1.1
├── minimist@0.0.5
├── mkdirp@0.3.5
├── stream-combiner@0.0.4 (duplexer@0.1.1)
└── concat-stream@1.0.1 (bops@0.0.6)
```

```
[stephen:/tmp]$ npm install wit@1.3.3
npm ERR! Linux 3.17.4-1-ARCH
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "wit@1.3.3"
npm ERR! node v0.10.33
npm ERR! npm  v2.1.12
npm ERR! path /tmp/node_modules/wit/bin/cmd.js
npm ERR! code ENOENT
npm ERR! errno 34

npm ERR! enoent ENOENT, chmod '/tmp/node_modules/wit/bin/cmd.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /tmp/npm-debug.log
```

**Comment by substack** [comment]
Thanks for catching that, it was a recurring npm issue I've run into. Fixed in 1.3.4.

**Comment by noffle** [comment]
Out of curiosity, what was the problem? Does npm not like the `1.3.3` magic number?
